packages/paste-core/components/select/__tests__/optiongroup.test.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from 'react';
 import {render} from 'react-dom';
-import {render as testRender} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {render as testRender, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
 // @ts-ignore typescript doesn't like js imports
 import axe from '../../../../../.jest/axe-helper';
 import {OptionGroup, Option} from '../src';
 import {createAttributeMap} from '../test-utils';
 
+expect.extend(matchers);
+
 interface MockOptionGroupProps extends React.OptgroupHTMLAttributes<HTMLOptGroupElement> {
+  element?: string;
```

Comment:
Not anything technically significant. I decided to simply declare the `element` type here since the rest of the "mock" test component interfaces are defined that way. Do you think this creates a level of confusion?